com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DepthOfFieldCoC.compute

```diff
@@ -45,7 +47,7 @@ void KMainPhysical(uint3 dispatchThreadId : SV_DispatchThreadID)
 {
     UNITY_XR_ASSIGN_VIEW_INDEX(dispatchThreadId.z);
 
-    float depth = LoadCameraDepth(dispatchThreadId.xy);
+    float depth = CameraDepth(_DepthMinMaxAvg, dispatchThreadId.xy);
```

Comment:
Note I guess for me if this gets merged before TAAU, these coordinates for posSS are not valid as they assume output resolution is same as depth buffer res.